target/linux/ramips/image/mt7621.mk

```diff
@@ -1394,6 +1394,28 @@ define Device/xiaomi_mi-router-ac2100
 endef
 TARGET_DEVICES += xiaomi_mi-router-ac2100
 
+define Device/xiaomi_mi-router-cr6606
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
```

Comment:
The lzma loader is for the ancient u-boot 1.1.3 which can't properly extract a big kernel. The new u-boot 2018 from mediatek has proper lzma decompression support and lzma-loader isn't needed. This line can be dropped.

Reply:
Give a shot, got stuck like a FIT image while booting. Not sure why.

Reply:
You can't change your load address in mt7621.mk without a loader that can relocate itself (e.g. we have lzma-loader and relocate-kernel). This load address must match what the kernel makefile says when using fit or legacy lzma uimage.
Continue using lzma loader here then.

Reply:
Thanks, no idea about the build behavior before.
Tried relocate-kernel with both fit image and legacy image, still can't make it boot. The uboot reversion is 2018.09. Stick with lzma-loader for now.